CHANGELOG.md

```diff
@@ -1,24 +1,31 @@
-Changelog for grphp-statsd.
+#Changelog for grphp-statsd.
 
-h3. 0.1.1
+###1.0.0
```

Comment:
Please use `0.2`, in 0.x versions minor is considered "breaking", because the whole library is unstable.

Reply:
fixed